src/main/kotlin/app/revanced/patches/all/telephony/sim/spoof/SpoofSimCountryPatch.kt

```diff
@@ -0,0 +1,76 @@
+package app.revanced.patches.all.telephony.sim.spoof
+
+import app.revanced.patcher.extensions.InstructionExtensions.getInstruction
+import app.revanced.patcher.extensions.InstructionExtensions.removeInstruction
+import app.revanced.patcher.extensions.InstructionExtensions.replaceInstruction
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patcher.patch.options.types.StringPatchOption.Companion.stringPatchOption
+import app.revanced.patcher.util.proxy.mutableTypes.MutableMethod
+import app.revanced.util.patch.AbstractTransformInstructionsPatch
+import com.android.tools.smali.dexlib2.builder.instruction.BuilderInstruction11x
+import com.android.tools.smali.dexlib2.iface.ClassDef
+import com.android.tools.smali.dexlib2.iface.Method
+import com.android.tools.smali.dexlib2.iface.instruction.Instruction
+import com.android.tools.smali.dexlib2.iface.instruction.ReferenceInstruction
+import com.android.tools.smali.dexlib2.iface.reference.MethodReference
+import java.util.*
+
+
+@Patch(
+    name = "Spoof SIM country",
+    description = "Spoofs country information returned by the SIM card provider.",
+    use = false,
+)
+@Suppress("unused")
+object SpoofSimCountryPatch : AbstractTransformInstructionsPatch<Pair<Int, String>>() {
+    private val networkCountryIso by stringPatchOption(
+        "getNetworkCountryIso",
```

Comment:
The option key should not be abused to check for the reference name in `filterMap`.